packages/validate/src/ValidateMixin.js

```diff
@@ -1,663 +1,616 @@
+       * @return {FeedbackMessage[]}
        */
-      static __getLocalizeKeys(key, validatorName) {
-        const result = [];
-        this.localizeNamespaces.forEach(ns => {
-          const namespace = typeof ns === 'object' ? Object.keys(ns)[0] : ns;
-          result.push(`${namespace}+${validatorName}:${key}`);
-          result.push(`${namespace}:${key}`);
-        });
-        return result;
+      async __getFeedbackMessages(validators) {
+        let fieldName = await this.fieldName;
+        return Promise.all(
+          validators.map(async validator => {
+            if (validator.config.fieldName) {
+              fieldName = await validator.config.fieldName;
+            }
+            const message = await validator._getMessage({
+              modelValue: this.modelValue,
+              formControl: this,
+              fieldName,
+            });
+            return { message, type: validator.type, validator };
+          }),
+        );
       }
 
       /**
-       * type can be 'error', 'warning', 'info', 'success'
+       * @desc Responsible for retrieving messages from Validators and
+       * (delegation of) rendering them.
        *
-       * a Validator can be
-       * - special string
-       *     'required'
-       * - function e.g
-       *     MyValidate.isEmail, isCat, ...
-       * - array for parameters e.g.
-       *     [minMaxLength, {min: 10, max: 15}],
-       *     [minLength, {min: 5}],
-       *     [contains, 'thisString']
+       * For `._feedbackNode` (extension of LionValidationFeedback):
+       * - retrieve messages from highest prio Validators
+       * - provide the result to custom feedback node and let the
+       * custom node decide on their renderings
+       *
+       * In both cases:
+       * - we compute the 'show' flag (like 'hasErrorVisible') for all types
+       * - we set the customValidity message of the highest prio Validator
+       * - we set aria-invalid="true" in case hasErrorVisible is true
        */
-      validateType(type) {
-        const validators = this.getValidatorsForType(type);
-        if (!(validators && Array.isArray(validators) && validators.length > 0)) return;
-
-        const resultList = [];
-        let value = this.modelValue; // This will end up being modelValue or Unparseable.viewValue
-
-        for (let i = 0; i < validators.length; i += 1) {
-          const validatorArray = Array.isArray(validators[i]) ? validators[i] : [validators[i]];
-          let validatorFn = validatorArray[0];
-          const validatorParams = validatorArray[1];
-          const validatorConfig = validatorArray[2];
-
-          let isRequiredValidator = false; // Whether the current is the required validator
-          if (typeof validatorFn === 'string' && validatorFn === 'required' && this.__isRequired) {
-            validatorFn = this.__isRequired;
-            isRequiredValidator = true;
-          }
+      async _updateFeedbackComponent() {
```

Comment:
```js
// TODO: as _updateFeedbackComponent is called without await in updated and in an event it means that a getting
// the messageMap might take longer and a call afterwards who set's the value to nothing gets overwritten by the
// later resolve of the messageMap
// therefore we await here for an existing feedbackComplete however this is not ideal as it means we wait
// for messages which we then never display => use a que? handle promise/resolve/reject manually without await?
// feedbackNode should handle promises? maybe not
if (this.feedbackComplete) {
  await this.feedbackComplete;
}
```

=> 3 failed tests as we probably would now need to await 2 times in the tests... 
=> maybe already invest in a proper solution?
=> or do "above" hack and skip tests or "hack" them as well... and then release so we can move forward... and in parallel start working on this